tests/release/pipelines/fbc_release.go

```diff
@@ -173,6 +180,58 @@ var _ = framework.ReleasePipelinesSuiteDescribe("FBC e2e-tests", Label("release-
 			})
 		})
 	})
+
+	Describe("with FBC pre-GA process", Label("fbcPreGAfix"), func() {
+		var component *appservice.Component
+
+		BeforeAll(func() {
+			devFw, err = framework.NewFrameworkWithTimeout(
+				devWorkspace,
+				time.Minute*60,
+				stageOptions,
+			)
+			Expect(err).NotTo(HaveOccurred())
+
+			managedFw, err = framework.NewFrameworkWithTimeout(
+				managedWorkspace,
+				time.Minute*60,
+				stageOptions,
+			)
+			Expect(err).NotTo(HaveOccurred())
+
+			_, err = devFw.AsKubeDeveloper.HasController.CreateApplication(fbcPreGAAppName, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+
+			_, err = devFw.AsKubeDeveloper.ReleaseController.CreateReleasePlan(fbcPreGARPName, devNamespace, fbcPreGAAppName, managedNamespace, "true")
+			Expect(err).NotTo(HaveOccurred())
+
+			createFBCReleasePlanAdmission(fbcPreGARPAName, *managedFw, devNamespace, managedNamespace, fbcPreGAAppName, fbcPreGAECPolicyName, relSvcCatalogPathInRepo, "true", issueId, productName, productVersion)
+			component = releasecommon.CreateComponentByCDQ(*devFw, devNamespace, managedNamespace, fbcPreGAAppName, fbcPreGACompName, fbcSourceGitURL)
+			createFBCEnterpriseContractPolicy(fbcPreGAECPolicyName, *managedFw, devNamespace, managedNamespace)
+		})
+
+		AfterAll(func() {
+			if !CurrentSpecReport().Failed() {
+				Expect(devFw.AsKubeDeveloper.HasController.DeleteApplication(fbcPreGAAppName, devNamespace, false)).NotTo(HaveOccurred())
+				Expect(managedFw.AsKubeDeveloper.TektonController.DeleteEnterpriseContractPolicy(fbcPreGAECPolicyName, managedNamespace, false)).NotTo(HaveOccurred())
+				Expect(managedFw.AsKubeDeveloper.ReleaseController.DeleteReleasePlanAdmission(fbcPreGARPAName, managedNamespace, false)).NotTo(HaveOccurred())
+			}
+		})
+
+		var _ = Describe("FBC pre-GA post-release verification", func() {
+			It("verifies that a build PipelineRun is created in dev namespace and succeeds", func() {
+				assertBuildPipelineRunCreated(*devFw, devNamespace, managedNamespace, fbcPreGAAppName, component)
+			})
+
+			It("verifies the fbc release pipelinerun is running and succeeds", func() {
```

Comment:
I guess if it succeeds, it is not running anymore.

Reply:
Or does this function somehow watches the pipelinerun until it finishes?

Reply:
Yes. this [function](https://github.com/redhat-appstudio/e2e-tests/blob/f9d5f1838d77ad4992d8272aa885e353fecdc074/tests/release/pipelines/fbc_release.go#L244) watches the pipelinerun until it finishes .

Reply:
I see. Thanks!